src/RA_StringUtils.h

```diff
@@ -177,52 +177,33 @@ class StringBuilder
     template<typename T>
     void Append(const T& arg)
     {
-        PendingString pPending;
         if (m_bPrepareWide)
-        {
-            pPending.WString = ra::ToWString(arg);
-            pPending.DataType = PendingString::Type::WString;
-        }
+            m_vPending.emplace_back(std::wstring{ra::ToWString(arg)});
```

Comment:
This might seem unnecessary but because the `ToWString` and `ToString` methods are const, it has to be converted to `rvalue` reference (a non-owning temporary).